docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,197 @@
+ Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit https://random-word-3421.edgestack.me to access your Edge Stack installation and for additional configuration.
+ ```
 
-    ```bash
-    (⎈ |minikube:ambassador)$ minikube service list
-    |-------------|------------------|--------------------------------|
-    |  NAMESPACE  |       NAME       |              URL               |
-    |-------------|------------------|--------------------------------|
-    | ambassador  | ambassador       | http://192.168.64.2:31230      |
-    |             |                  | http://192.168.64.2:31042      |
-    | ambassador  | ambassador-admin | No node port                   |
-    | ambassador  | ambassador-redis | No node port                   |
-    | default     | kubernetes       | No node port                   |
-    | kube-system | kube-dns         | No node port                   |
-    |-------------|------------------|--------------------------------|
-    ```
+Your new [Edge Policy Console](/about/edge-policy-console) will open automatically in your browser at the provided URL.
 
-    Use any of the URLs listed next to `ambassador` to access the Ambassador Edge Stack.
+### Minikube Users
 
-3. Navigate to `http://<your-IP-address>` and click through the certificate warning for access to the Edge Policy Console interface. The certificate warning appears because, by default, the Ambassador Edge Stack uses a self-signed certificate for HTTPS.
-    * Chrome users should click **Advanced > Proceed to website**.
-    * Safari users should click **Show details > visit the website** and provide your password.
+If you're a Minikube user, Edge Control will not be able to provide a domain name. However, you will still be given an IP address to access the Edge Policy Console.
 
-4. To login to the [Edge Policy Console](../../about/edge-policy-console), download and install `edgectl`, the command line tool Edge Control, by following the provided instructions on the page. The Console lists the correct command to run and provides download links for the `edgectl` binary.
+Edge Control will automatically configure TLS **if your cluster is publicly accessible.**
 
-  The Edge Policy Console must authenticate your session using a Kubernetes Secret in your cluster. Edge Control accesses that secret using `kubectl`, then sends a URL to your browser that contains the corresponding session key. This extra step ensures that access to the Edge Policy Console is just as secure as access to your Kubernetes cluster.
+**To get started:**
 
-  For more information, see [Edge Control](../../reference/edge-control).
+1. Ensure your cluster is publicly accessible by following these directions: `url to documentation page about how to make your cluster publicly accessible`
```

Comment:
I still say this line makes no sense for the Minikube instructions. Practically speaking, you should not make your cluster publicly accessible if you're running Minikube.